src/components/Documentation.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <section class="section" id="dochome">
+    <div class="columns is-mobile is-centered">
+      <div class="column is-three-quarters-tablet is-half-desktop has-text-centered">
+        <div class="block">
+          <h1 class="title is-1">Documentation</h1>
+          <p>Resources for using Pangeo Forge</p>
+        </div>
+      </div>
+    </div>
+    <div class="content" id="firststeps">
+        <h3 class="title is-3">First steps</h3>
+        <p>New to Pangeo Forge?  Start here!</p>
+        <ul>
+            <li><strong>What is Pangeo Forge?</strong> - Read more about what Pangeo Forge is and
+            how it works.</li>
+            <li><strong>Introduction Tutorial</strong> - Ready to code?  Walk through creating and
```

Comment:
Should this link to https://pangeo-forge.readthedocs.io/en/latest/tutorials/index.html ?

I haven't reviewed the content of those tutorials recently and don't know how representative/accessible they are.

Reply:
Those tutorials are nice, but I think the goal of an introduction tutorial is specific - an intro tutorial should walk a brand new person step by step through the simplest possible meaningful version of deploying a data transformation.  I see that as a new tutorial and I'm not sure right now if I think it should live as a `pangeo-forge-recipes` tutorial or a Pangeo Forge-wide tutorial.  Conceptually I want it to be the latter, but I'm not sure if there is enough content for that.  I discuss more in my comment below.

Reply:
Updated to a link to the Introduction Tutorial section of the readthedocs page that was created in [the most recent PR](https://github.com/pangeo-forge/pangeo-forge-recipes/pull/178#event-5164046212).